tst/CTA.WebForms2Blazor.Tests/FileConverters/ViewFileConverterTests.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CTA.WebForms2Blazor.ControlConverters;
+using CTA.WebForms2Blazor.FileConverters;
+using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Services;
+using NUnit.Framework;
+
+namespace CTA.WebForms2Blazor.Tests.FileConverters
+{
+    [TestFixture]
+    public class ViewFileConverterTests
+    {
+        [Test]
+        public async Task TestViewFileConverter()
+        {
+            var controlMap = new Dictionary<string, ControlConverter>() {["asp:hyperlink"]=new HyperLinkControlConverter()};
+            FileConverter fc = new ViewFileConverter(FileConverterSetupFixture.TestProjectPath, FileConverterSetupFixture.TestViewFilePath, controlMap);
+            
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.Single();
+            
+            byte[] bytes = fi.FileBytes;
+            var projectFileContents = Encoding.UTF8.GetString(bytes);
+            
+            string newPath = Path.Combine(FileConverterSetupFixture.TestFilesDirectoryPath, "testing.razor");
+            string relativePath = Path.GetRelativePath(FileConverterSetupFixture.TestProjectPath, newPath);
+
+            Assert.True(projectFileContents.Contains("href"));
+            Assert.True(projectFileContents.Contains("class"));
+            Assert.False(projectFileContents.Contains("asp:HyperLink"));
+            Assert.False(projectFileContents.Contains("CssClass"));
+            Assert.False(projectFileContents.Contains("NavigateUrl"));
+            Assert.IsTrue(fi.RelativePath.Equals(relativePath));
+        }
```

Comment:
Couple of suggestions: 

1. Can we rename this test to `HyperLinkControlConverter_Returns_Href_Node()`? 
2. Let's rename `projectFileContents` to view fileContents
3. We can replace the assert statements with a single assert where we check that the file contents are equal to some string literal. This test would be more robust and more clearly show what the expected output would be (allows the test to serve as documentation)

Reply:
I think the 3rd point might be a bit hard, since when other features are added the returned contents will be different from what it is currently since other nodes will be converted. I can change this test to make it more unit based though